scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/descriptor/JobDescriptorImpl.java

```diff
@@ -83,14 +84,14 @@
 
     /** Job tasks to be able to be schedule */
     @XmlTransient
-    private Map<TaskId, EligibleTaskDescriptor> eligibleTasks = new ConcurrentHashMap<>();
+    public Map<TaskId, EligibleTaskDescriptor> eligibleTasks = new ConcurrentHashMap<>();
 
     /** Those are not directly eligible, and will be triggered by an IF control flow action */
     @XmlTransient
     private Map<TaskId, EligibleTaskDescriptor> branchTasks = new ConcurrentHashMap<>();
 
     /** Job running tasks */
-    private Map<TaskId, TaskDescriptor> runningTasks = new ConcurrentHashMap<>();
+    public Map<TaskId, TaskDescriptor> runningTasks = new ConcurrentHashMap<>();
```

Comment:
Do we need this Map to be public? 

What about the @XmlTransient like at the eligibleTasks ?